RoadMap_VK/RoadMap_VK/Model/Post.swift

```diff
@@ -0,0 +1,32 @@
+// Post.swift
+// Copyright © RoadMap. All rights reserved.
+
+/// Публикации
```

Comment:
```suggestion
/// Посты
```
Post же название у файла

Reply:
публикация это перевод слова Post